drivers/wifi/nrf700x/zephyr/src/zephyr_fmac_main.c

```diff
@@ -53,7 +53,7 @@ BUILD_ASSERT(RPU_PKTRAM_SIZE >=
 static const unsigned char aggregation = 1;
 static const unsigned char wmm = 1;
 static const unsigned char max_num_tx_agg_sessions = 4;
-static const unsigned char max_num_rx_agg_sessions = 2;
+static const unsigned char max_num_rx_agg_sessions = 8;
```

Comment:
is this related?

Reply:
Yeah, apparently required for the certification test. Details in https://projecttools.nordicsemi.no/jira/browse/SHEL-1269